client/packages/common/src/ui/layout/tables/components/Cells/NumberInputCell/PositiveNumberInputCell.tsx

```diff
@@ -27,6 +27,7 @@ export const PositiveNumberInputCell = <T extends RecordWithId>({
       error={isError}
       value={buffer}
       onChange={newValue => {
+        if (newValue === undefined) return;
```

Comment:
Again, seems appropriate to prevent clearing the value, as it's used in a table, and it starts with a value from the database (and could be 0).